skimage/color/colorconv.py

```diff
@@ -1456,3 +1480,199 @@ def _prepare_lab_array(arr):
     if shape[-1] < 3:
         raise ValueError('Input array has less than 3 color channels')
     return dtype.img_as_float(arr, force_copy=True)
+
+def rgb2yuv(rgb):
+    """RGB to YUV color space conversion.
+
+    Parameters
+    ----------
+    rgb : array_like
+        The image in RGB format, in a 3- or 4-D array of shape
+        ``(.., ..,[ ..,] 3)``.
+
+    Returns
+    -------
+    out : ndarray
+        The image in YUV format, in a 3- or 4-D array of shape
+        ``(.., ..,[ ..,] 3)``.
+
+    Raises
+    ------
+    ValueError
+        If `rgb` is not a 3- or 4-D array of shape ``(.., ..,[ ..,] 3)``.
+
+    Notes
+    -----
+    Y is between 0 and 1.  Use YCbCr instead of YUV for the color space which is commonly used by video codecs (where Y ranges from 16 to 235)
+    """
+    return _convert(yuv_from_rgb, rgb)
+
+def rgb2yiq(rgb):
+    """RGB to YIQ color space conversion.
+
+    Parameters
+    ----------
+    rgb : array_like
+        The image in RGB format, in a 3- or 4-D array of shape
+        ``(.., ..,[ ..,] 3)``.
+
+    Returns
+    -------
+    out : ndarray
+        The image in YIQ format, in a 3- or 4-D array of shape
+        ``(.., ..,[ ..,] 3)``.
+
+    Raises
+    ------
+    ValueError
+        If `rgb` is not a 3- or 4-D array of shape ``(.., ..,[ ..,] 3)``.
+    """
+    return _convert(yiq_from_rgb, rgb)
+
+def rgb2ypbpr(rgb):
+    """RGB to YIQ color space conversion.
+
+    Parameters
+    ----------
+    rgb : array_like
+        The image in RGB format, in a 3- or 4-D array of shape
+        ``(.., ..,[ ..,] 3)``.
+
+    Returns
+    -------
+    out : ndarray
+        The image in YIQ format, in a 3- or 4-D array of shape
+        ``(.., ..,[ ..,] 3)``.
+
+    Raises
+    ------
+    ValueError
+        If `rgb` is not a 3- or 4-D array of shape ``(.., ..,[ ..,] 3)``.
+    """
+    return _convert(ypbpr_from_rgb, rgb)
+
+def rgb2ycbcr(rgb):
+    """RGB to YCbCr color space conversion.
+
+    Parameters
+    ----------
+    rgb : array_like
+        The image in RGB format, in a 3- or 4-D array of shape
+        ``(.., ..,[ ..,] 3)``.
+
+    Returns
+    -------
+    out : ndarray
+        The image in YCbCr format, in a 3- or 4-D array of shape
+        ``(.., ..,[ ..,] 3)``.
+
+    Raises
+    ------
+    ValueError
+        If `rgb` is not a 3- or 4-D array of shape ``(.., ..,[ ..,] 3)``.
+
+    Notes
+    -----
+    The destination color space is sometimes also called "YUV"
```

Comment:
This is confusing. It might be better to make the note explicit, like you did above: `Y is between 0 and 1.  Use YCbCr instead of YUV for the color space which is commonly used by video codecs (where Y ranges from 16 to 235)`. Vise versa, of course